examples/ios/src/lib.rs

```diff
@@ -87,6 +89,65 @@ fn setup_scene(
         transform: Transform::from_xyz(-2.0, 2.5, 5.0).looking_at(Vec3::ZERO, Vec3::Y),
         ..default()
     });
+
+    // Test ui
+    commands.spawn_bundle(UiCameraBundle::default());
```

Comment:
This example isn't too large so it might not be useful but should this be split into two examples? 

First example focuses on teaching what is needed to get anything running on iOS, second example focuses on showcasing how to do UI in iOS?
2.

Reply:
It's the only iOS example and iOS examples require quite some extra files: 
- build_rust_deps.sh
- Makefile
- its own crate files (src, Cargo.toml, etc)
- a Xcode project folder and files

So I'm not sure if adding all that again would be worth it. Not against it though

Reply:
Yeah, I agree. Let's not create extra clutter until it's absolutely necessary :)